contrib/fuzz/docker_fuzzer.go

```diff
@@ -0,0 +1,78 @@
+// +build gofuzz,!windows
```

Comment:
Why do we need `!windows`? https://github.com/containerd/containerd/blob/main/remotes/docker/fetcher.go is platform-agnostic and existing fuzzers don't exclude Windows. https://github.com/containerd/containerd/tree/main/contrib/fuzz

Reply:
The reason I added this was that a windows test failed in the CI.

Reply:
Do you mean OSS-fuzz's CI or containerd's CI?

Reply:
containerd's

Reply:
Do you remember the error you got? I'm inclined to fix rather than ignoring.

Reply:
Sorry no, but the fuzzer is similar to how it was when the previous test failed, and now it passes.

Reply:
Yay! 💯